### DISCUSSION TITLE
Add an option for search with querystring

### DISCUSSION THREAD

**Discussion by ufoloko** [openstore-ecommerce/OpenStore#288] [Ideas]
Last version works sending an ajax call to set the search conditions and then redirect to products page to show results.

Should be great to have an option for sending a querystring parameter to run the search ? For example www.domain.com/Products?Search=textToSearch

This can be helpful to redirect searching for external links.

**Comment by DNNMonster** [comment]
There is a feature branch that tries to implement this.  It should have some other eyes review it before it gets merged into develop.  If you are compiling and have the chance to test it that'd be great.

https://github.com/openstore-ecommerce/OpenStore/tree/feature/288-first-render-search-query-support

**Reply by ufoloko** [reply]
Thanks for your response. I will prepare my development environment and try to compile. I will let you know how is working.
